spec/cddl/many.cddl

```diff
@@ -85,6 +90,12 @@ response = COSE_Sign<{
     ; Attributes. An optional list of response attributes defined in the spec. See
     ; the list of all attributes in this repo.
     ? 8 => [ * attribute ],
+
+    ; Proof.  If requested, this is the set of steps constituting a proof of existence
+    ; of what was requested in the state of the Merkle tree.
+    ; See https://github.com/liftedinit/merk/blob/develop/docs/algorithms.md#binary-format for
```

Comment:
I'd like to put the thoughts of this document into a separate specification document and not rely on implementation specific documentation.

Reply:
OK, I can put the binary structure into a separate adoc